packages/next/src/server/stream-utils/uint8array-helpers.ts

```diff
@@ -57,3 +57,26 @@ export function removeFromUint8Array(a: Uint8Array, b: Uint8Array) {
     return a
   }
 }
+
+/**
+ * Replace Uint8Array `search` with Uint8Array `replacement` in Uint8Array `target`.
+ * the tagIndex is the index of the `search` in `target`.
+ */
+export function replaceInUint8Array(
+  target: Uint8Array,
+  search: Uint8Array,
+  replacement: Uint8Array,
+  tagIndex: number
```

Comment:
this function sounds generic but this argument's name isn't. Maybe it should be renamed to `index`?

Reply:
renamed to `indexOfTarget`, cause unlike the other util, this util requires you got the index of `target` in `search` to avoid another round of string search and can speed up the operation.